docs/pages/attributesJSON.rst

```diff
@@ -46,9 +46,125 @@ Below are the supported JSON tags for Physics Manager Attributes templates, and
 	- list of strings
 	- A list of locations to query for supported object files that should be available to be loaded into the world.
 
+
+`Scene Dataset Attributes`_
+===========================
+Scene Dataset Attributes describe the names and file locations of all the assets and configuration files that make up a Scene Dataset. They are designed as a kind of index for a scene dataset, organizing the links to all the constituent components of the dataset into appropriate thematic categories, as well as providing an easy mechanism for customizing any of the configurations, all in a single file. All file paths referenced in this configuration should be relative to the directory where this Scene Dataset configuration resides.
+
+These file names should be formatted as follows:
+
+ 	<scenedatasetname>.scene_dataset_config.json
+
+`An example of an appropriately configured Scene Dataset Configuration file can be found below <../../../data/test_assets/dataset_tests/test_dataset.scene_dataset_config.json>`_:
+
+.. include:: ../../data/test_assets/dataset_tests/test_dataset.scene_dataset_config.json
+    :code: json
+
+'Scene Dataset: Stages, Objects, Light Setups and Scene Instances'
+------------------------------------------------------------------
+
+The Scene Dataset Attributes configuration consists of a hierarchy of JSON components used to specify all the component assets and attributes used within a scene dataset. Tags "stages", "objects", "light_setups" and "scene_instances" reference JSON objects that list the included assets and configurations for the specified type of construction. The JSON objects referenced by each of these 4 tags support the following tags:
```

Comment:
Same thought as above -- we should define what a stage is and what a "scene instance" is.